flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/ScalarOperatorGens.scala

```diff
@@ -1767,6 +1767,56 @@ object ScalarOperatorGens {
     }
   }
 
+  /**
+   * Return null when at least one of the elements is null.
```

Comment:
This explains only an edge case of the function generated here, but not really what is being generated. I think we should properly document what this method does.